routes/curations.js

```diff
@@ -44,8 +45,23 @@ router.patch(
   '/:type/:provider/:namespace/:name/:revision',
   asyncMiddleware(async (request, response) => {
     const coordinates = utils.toEntityCoordinatesFromRequest(request)
+    const token = request.app.locals.config.curation.store.github.token
+    const serviceGithub = Github.getClient({ token })
```

Comment:
seems like we should be setting up the serviceGitHub client somewhere near `request.app.locals.user.github.client` on the request using the same middleware as for the user client. Then we are just using it here.

Reply:
This is not being called anymore. This was the old patch function.

Reply:
this should be deleted. if it no longer used